testing/cmake/write_version-leading-zeroes/CMakeLists.txt

```diff
@@ -0,0 +1,25 @@
+#=============================================================================
+# Copyright (c) 2018-2021, NVIDIA CORPORATION.
```

Comment:
The most nitpicky nitpick: All these copyright header years should be only 2021 I think

Reply:
I followed the pattern that existed in cudf and raft where a copyright range existed. But at the minimum the copyright range is wrong ( should be 2021-2021? ), or do we change it next year to 2021-2022?

Reply:
https://github.com/rapidsai/rapids-cmake/issues/28